src/libs/timestamp-tree-links.js

```diff
@@ -0,0 +1,55 @@
+import select from 'select-dom';
+import {h} from 'dom-chef';
+import {getOwnerAndRepo} from './page-detect';
+import * as icons from './icons';
+
+const {ownerName, repoName} = getOwnerAndRepo();
+
+const getSHABeforeTimestamp = async timestamp => {
+	const url = `https://api.github.com/repos/${ownerName}/${repoName}/commits?until=${timestamp}&per_page=1`;
```

Comment:
If we’re positive we can’t do this without the API, we still need to support `/api` instead of the subdomain, for GHE.  You might want to pack it up as a `getApiUrl` function because we’ll probably need the same logic again in the future.